core/bessctl.cc

```diff
@@ -534,6 +534,9 @@ class BESSControlImpl final : public BESSControl::Service {
     }
 
     for (; i <= wid_filter; i++) {
+      if (workers[i] == nullptr) {
```

Comment:
Oh, that's right. Good catch. I'll rework `ListTcs()` to appropriately handle this case